client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -100,10 +100,29 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 
 		// Some Utils
 		handleSubmitForm = event => {
+			const { dirtyFields, fields, trackTracksEvent } = this.props;
+
 			if ( ! event.isDefaultPrevented() && event.nativeEvent ) {
 				event.preventDefault();
 			}
 
+			forEach( dirtyFields, function( value ) {
```

Comment:
Any good reason to use `forEach` here? I think it's generally slower, how about using `map` or something similar here?